doc/jvm/index.rst

```diff
@@ -38,6 +38,7 @@ Contents
   XGBoost4J-Spark-GPU Tutorial <xgboost4j_spark_gpu_tutorial>
```

Comment:
Do we need to change this tutorial? In the offline review, there's a breaking change for the jar names.

Reply:
I will have a following PR for the whole jvm doc updating.

Reply:
It's Done, Mark as resolved.